app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,23 @@
   </head>
 
   <body>
+  <p class="navbar-text float-right">
```

Comment:
wrong indentation, this should never happen.

Reply:
Updated